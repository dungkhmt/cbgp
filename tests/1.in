20 20
20 49
8 19
6 5
3 7
19 5
5 2
5 10
8 5
15 19
8 2
2 3
11 13
9 1
12 12
12 14
18 10
5 11
16 4
6 18
13 5
10 17
5 3
2 2
11 10
12 2
2 7
2 4
16 3
19 10
8 15
20 1
1 2
9 20
12 20
2 20
2 16
10 6
