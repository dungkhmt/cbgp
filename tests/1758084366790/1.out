(16, 12), (8, 15), (10, 6), (14, 18), (5, 1), (2, 1), (0, 13), (2, 0), (6, 5), (3, 2), (3, 5), (14, 11), (14, 0), (11, 3), (4, 15), (3, 8), (7, 5), (11, 19), (8, 19), (6, 2), (11, 8), (14, 3), (9, 13), (4, 0), (3, 18), (0, 11), (3, 6), (9, 19), (0, 1), (0, 8), (7, 17), (17, 5), (18, 11), (10, 3), (0, 15), (0, 16), (0, 5), (12, 1), (4, 8), (14, 2), (2, 7), (0, 9), (3, 0), (4, 3), (0, 19), 
numberIntersection=242 
minAngle=0.0 
sumAngle=6.118358264482202 
minEdgeLength=10.0 
sumEdgeLength=1776.6527422898803 
minDistance=0.0 
sumDistance=20958.533538348533 

-242.0000000000 0.0000000000 6.1183582645 10.0000000000 1776.6527422899 0.0000000000 20958.5335383485
0: (30, 40),
1: (40, 40),
2: (10, 57),
3: (40, 48),
4: (40, 58),
5: (10, 23),
6: (40, 80),
7: (24, 80),
8: (9, 80),
9: (0, 66),
10: (0, 52),
11: (0, 44),
12: (0, 36),
13: (0, 28),
14: (0, 14),
15: (9, 0),
16: (24, 0),
17: (40, 0),
18: (40, 22),
19: (40, 32),
30 40 40 40 10 57 40 48 40 58 10 23 40 80 24 80 9 80 0 66 0 52 0 44 0 36 0 28 0 14 9 0 24 0 40 0 40 22 40 32 0: (30, 40), 1: (40, 40), 2: (10, 57), 3: (40, 48), 4: (40, 58), 5: (10, 23), 6: (40, 80), 7: (24, 80), 8: (9, 80), 9: (0, 66), 10: (0, 52), 11: (0, 44), 12: (0, 36), 13: (0, 28), 14: (0, 14), 15: (9, 0), 16: (24, 0), 17: (40, 0), 18: (40, 22), 19: (40, 32), 
Time taken: 0.035 s
